### DISCUSSION TITLE
Welcome to ISSM Discussions!

### DISCUSSION THREAD

**Discussion by TidbitSoftware** [ISSMteam/ISSM#10] [Announcements]
## 👋 Welcome!
Please use Discussions...
  * to ask questions about specific use cases of ISSM (e.g. how to represent a certain process in the model).
  * for questions about configuring ISSM or external packages.
  * to connect and collaborate with other ISSM users.

For bugs related to configuring, compiling, or running ISSM, please [submit an Issue](https://github.com/ISSMteam/ISSM/issues).

NOTE: We will be disabling the existing [ISSM Forum](https://issm.ess.uci.edu/forum/) on August 27th, at which point we will be using Discussions and Issues exclusively. If there are any helpful discussions on the ISSM Forum that you think belong here, please let us know or feel free to add them yourself.

**Comment by Judson4** [comment]
I am looking for a free vulnerability scan option for an air gapped standalone system. I have tried Nmap but not having any luck getting accurate scans.

**Reply by inwoo-park** [reply]
Spam...?!
